src/highlight/HighlightCanvas.tsx

```diff
@@ -5,11 +5,12 @@ import './HighlightCanvas.scss';
 
 export type Props = {
     activeId: string | null;
-    annotations: AnnotationHighlight[];
+    annotations: Pick<AnnotationHighlight, 'id' | 'target'>[];
```

Comment:
What problem are we trying to solve here?

Reply:
When rendering the staged highlight, it doesn't have all the other properties of an `AnnotationHighlight` but to reuse `HighlightCanvas` and elsewhere it expects the least common denominator of `id` and `target`